core/build.sbt

```diff
@@ -334,161 +331,150 @@ lazy val operator =
     .dependsOn(blueprint % "compile->compile;test->test")
     .settings(
       libraryDependencies ++= Vector(
-        AkkaSlf4j,
-        AkkaStream,
-        Ficus,
-        Logback,
-        Skuber,
-        AkkaStreamTestkit,
-	JacksonDatabind,
-        ScalaTest,
-        ScalaCheck              % "test",
-        Avro4sJson              % "test",
-      )
+            AkkaSlf4j,
+            AkkaStream,
+            Ficus,
+            Logback,
+            Skuber,
+            AkkaStreamTestkit,
+            JacksonDatabind,
+            ScalaTest,
+            ScalaCheck % "test",
+            Avro4sJson % "test"
+          )
     )
     .settings(
       scalaVersion := "2.12.9",
       crossScalaVersions := Vector(scalaVersion.value),
       organization := "com.lightbend.cloudflow",
-
       skip in publish := true,
-
       mainClass in Compile := Some("cloudflow.operator.Main"),
-
       publishArtifact in (Compile, packageDoc) := false,
       publishArtifact in (Compile, packageSrc) := false,
-
       buildOptions in docker := BuildOptions(
-        cache = true,
-        removeIntermediateContainers = BuildOptions.Remove.OnSuccess,
-        pullBaseImage = BuildOptions.Pull.IfMissing
-        // TODO: "Always" won't work unless you have used `docker login` with
-        // your own service/user account. We should move to use some
-        // way to call `gcloud docker --` instead.
-        // pullBaseImage = BuildOptions.Pull.Always
-      ),
-
+            cache = true,
+            removeIntermediateContainers = BuildOptions.Remove.OnSuccess,
+            pullBaseImage = BuildOptions.Pull.IfMissing
+            // TODO: "Always" won't work unless you have used `docker login` with
+            // your own service/user account. We should move to use some
+            // way to call `gcloud docker --` instead.
+            // pullBaseImage = BuildOptions.Pull.Always
+          ),
       imageNames in docker := Seq(
-        ImageName(
-          registry = None,
-          namespace = Some("lightbend"),
-          repository = "cloudflow-operator",
-          tag = Some(cloudflowBuildNumber.value.asVersion)
-        )
-      ),
-
+            ImageName(
+              registry = None,
+              namespace = Some("olofwalker"),
```

Comment:
This isn't meant to be committed, right?

Reply:
No, have to clean this PR up a bit, was added in a hurry yesterday evening.